### PR TITLE
airbyte-ci: add supportsFileTransfer to metadata

### PR DESCRIPTION
**Pull request by stephane-airbyte**
we're adding a new field to a connector's metadata for universal file transfer. The field is already supported by the platform, and there's a destination connector (s3) that is using it in a pending PR. I _think_ this is enough, but I'm not sure. Please let me know if there's another system that needs similar changes

### CODE REVIEW

**Comment by stephane-airbyte** [comment]
* **#47007** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/47007?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stephane-airbyte and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by natikgadzhi** [comment]
@aldogonzalez8 that works with your PR for file transfers as well?

**Comment by aldogonzalez8** [comment]
> @aldogonzalez8 that works with your PR for file transfers as well?

I will. I tried to add it, but it throws some build or platform exceptions; I think I was missing all the changes that @stephane-airbyte is doing for the destination, so I will copy-paste/learn for source definitions.

**Review by benmoriceau — APPROVED** [review]

File: airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectorRegistryDestinationDefinition.yaml
Comment (alafanechere) [inline airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectorRegistryDestinationDefinition.yaml:1]: Let's add it to `ConnectorRegistrySourceDefinition.yaml`

**Review by alafanechere — COMMENTED** [review]
Can you please run `poetry lock` in the `orchestrator` package and bump its version too? It will make the registry generation logic use your new model.

**Comment by alafanechere** [comment]
>  I think this is enough, but I'm not sure. 

@stephane-airbyte it should be enough to expose your new field on our[ oss/cloud registry](https://connectors.airbyte.com/files/registries/v0/cloud_registry.json).
You might have to do some platform-side change add this field to the actor definition model.

**Review by alafanechere — APPROVED** [review]
LGTM, well done @stephane-airbyte

**Pull request merged**